tests/console/verify_bcache_partitioning.pm

```diff
@@ -0,0 +1,57 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate bcache in writethrough
+# Usually the caching device is a fast device (ssd). In this scenario a rotational device is used.
+# The whole second disk is used as a caching device with /home mounted as a backing partition.
+# This doesnt give any significant performance advantages but it is better to test bcache with this setup
+# than use bcache without caching(which yast provides a option if we want to do so)
+# Scenarios covered:
+# - Verify that certain values are set correctly in a bcache setup after installation.
+# - Write operation in a backing device can be performed and validate that cache is working watching the cache hits.
+# Maintainer: yiannis bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler qw(get_test_suite_data get_data_value_from);
+use Test::Assert 'assert_true';
+
+sub run {
+    my $test_data  = get_test_suite_data();
+    my $cachingdev = get_data_value_from($test_data->{profile}->{partitioning}->[1]->{drive}->{device});
```

Comment:
I wonder myself if we should use test data for each module that we validate, the problem to reuse it here is that the expression is pretty long and the order could change, to use indexes is always a bad idea :) I would create separate key for this data so it would be more readable. But leave it as it is if you prefer.

Reply:
see above